tests/unit/test_mixins.py

```diff
@@ -377,6 +373,85 @@ def test_is_admin_grader(self, preferences, expected_is_admin_grader):
 
 
 @ddt.ddt
+class TestAuthXBlockMixin(TestCase, TestWithPatchesMixin):
+
+    USER_ID = 1234
+    COURSE_ID = 4321
+
+    class AuthXBlockMixinGuineaPig(AuthXBlockMixin):
+
+        @property
+        def course_id(self):
+            return 4321
```

Comment:
@e-kolpakov This si one place where Python scoping doesn't work well. ``TestAuthXBlockMixin.COURSE_ID`` is not accessible in `AuthXBlockMixinGuineaPig` namespace.